source/enumerate-indexes.rst

```diff
@@ -12,8 +12,8 @@ Enumerating Indexes
 :Status: Draft
 :Type: Standards
 :Server Versions: 1.8-2.7.5, 2.8.0-rc3 and later
-:Last Modified: January 15, 2015
-:Version: 0.3
+:Last Modified: 2017-09-19
+:Version: 0.3.1
```

Comment:
I think this should be 0.4, but I do not feel strongly about that. My argument is that this isn't a bugfix to the spec, it's a significant improvement.